### DISCUSSION TITLE
ECU-R no IP address on my router

### DISCUSSION THREAD

**Discussion by TWE0GEE** [ksheumaker/homeassistant-apsystems_ecur#201] [General]
When I connect ECU via LAN than I can see an IP on my router, but in HA I can't connect that IP.
when I'm connected via LAN I used Packet Sender but I couldn't connect on port 8899.
I was able to connect to port 4540.

then I switched the ECU-R to Wifi, I connected the wifi in the ECU Manager app. 
But then I can't see an IP from the ECU-R on my router.

I also have installed HACS and downloaded APS several times as mentioned in other issues.

What am I doing wrong?

**Comment by HAEdwin** [comment]
For the first paragraph; this could be correct if you own an ECU-R starting with 2160xxxxxx
If you switch to WiFi and the green led with the cloud icon is on, you are connected to the internet and have to look in your router for a node starting with ESP. Just keep pinging this node and pull the plug on the ECU to make sure you have the right one. MAC address should start with something like d8:bf:c0 then that is the node you want. Best thing to do is to assign a fixed IP-address.

**Comment by TWE0GEE** [comment]
Hallo Edwin,

 

Dank voor de feedback ik heb de ECU nu in HA

 

Groet Tonnie

 

Van: HAEdwin ***@***.***> 
Verzonden: maandag 4 september 2023 13:19
Aan: ksheumaker/homeassistant-apsystems_ecur ***@***.***>
CC: TWE0GEE ***@***.***>; Author ***@***.***>
Onderwerp: Re: [ksheumaker/homeassistant-apsystems_ecur] ECU-R no IP address on my router (Discussion #201)

 

For the first paragraph; this could be correct if you own an ECU-R starting with 2160xxxxxx
If you switch to WiFi and the green led with the cloud icon is on, you are connected to the internet and have to look in your router for a node starting with ESP. Just keep pinging this node and pull the plug on the ECU to make sure you have the right one. MAC address should start with something like d8:bf:c0 then that is the node you want. Best thing to do is to assign a fixed IP-address.
